doc/manual/parameters.tex

```diff
@@ -3319,7 +3319,8 @@ \subsection{Parameters in section \tt Heating model/Constant heating}
 {\it Default:} 0e0
 
 
-{\it Description:} The specific rate of heating due to radioactive decay (or other bulk sources you may want to describe). This parameter corresponds to the variable $H$ in the temperature equation stated in the manual, and the heating term is $
ho H$. Units: W/kg.
+{\it Description:} The specific rate of heating due to radioactive decay (or other bulk sources you may want to describe). This parameter corresponds to the variable $H$ in the temperature equation stated in the manual, and the heating term is $
+ho H$. Units: W/kg.
```

Comment:
Could you please revert your changes to parameters.tex? They will be made automatically by the update_parameters.sh script every once in a while (so your plugin will show up in the manual in a while). In case you did these with the script, and feel it is important to have the documentation of your plugin in the manual immediately, please submit a separate pull request with the title 'Update parameters.tex' or similar. That way the changes are more organized into logical units :wink: